src/app/components/Curation/getComponentName/index.test.ts

```diff
@@ -30,7 +31,13 @@ describe('getComponentName', () => {
   `(
     'should return $expected when visual style is $visualStyle and visual prominence is $visualProminence',
     ({ visualStyle, visualProminence, expected }) => {
+      console.log(getComponentName(visualStyle, visualProminence));
       expect(getComponentName(visualStyle, visualProminence)).toBe(expected);
+      
     },
   );
+  // Include this test in the table above instead even though we are using the extra optional argument?
```

Comment:
Yes, I think it would be best to include this, to make it easier to add new scenarios in future. You'll need to add a new column to the table, and it will be `null` for all of the existing test scenarios.

Reply:
Done. I have put some comments in the getComponentName index file about also adding curationType as a common criteria to the other curation types. Thinking about it and might try it. Opinions welcome!

It is basically about adding curationType in here:

  const componentsByVisualStyleAndProminence = {
    [`${BANNER}_${MINIMUM}`]: NOT_SUPPORTED,
    [`${BANNER}_${LOW}`]: NOT_SUPPORTED,
    [`${BANNER}_${NORMAL}`]: MESSAGE_BANNER,
    [`${BANNER}_${HIGH}`]: NOT_SUPPORTED,
    [`${BANNER}_${MAXIMUM}`]: NOT_SUPPORTED,
    [`${NONE}_${NORMAL}`]: SIMPLE_CURATION_GRID,
    [`${NONE}_${HIGH}`]: HIERARCHICAL_CURATION_GRID,
    [`${COLLECTION}_${HIGH}`]: HIERARCHICAL_CURATION_GRID,
    [`${RANKED}_${NORMAL}`]: MOST_READ,
  };

as well as the option for RADIO_SCHEDULE. I think it might be a good idea as it keeps it consistent, and although it adds a load more combinations that could happen (and might need to put as NOT_SUPPORTED...) it would also stop something like BANNER NORMAL being input into tipo with a radio-schedule, which would cause it to return the component name as message banner but with radio schedule data, which seems like it could be a problem.

Reply:
I have put these changes in now and another comment about what I am thinking about for other combinations being added